direct/src/dist/commands.py

```diff
@@ -580,6 +582,12 @@ def run(self):
             if self.macos_main_app and 'macosx' in platform:
                 self.bundle_macos_app(build_dir)
 
+    def announce(self, msg, level=logging.INFO):
+        """ Override the command's built-in announce function as older versions of setuptools does not
+        use logging style levels.
+        """
+        build_apps.log.log(level, msg)
```

Comment:
Could you explain this?  Does this interact with the setuptools logging?  I would like not to be using two log systems at the same time.

Reply:
Yes, essentially setuptools includes the disttools logger, but they're monkey-patch the some functions in the distutil  version (https://github.com/pypa/setuptools/blob/main/setuptools/logging.py#L27) to use the builtin logging module. In other places they use the logging module but with the log levels divided by 10. (so WARNING would become WARNING/10), I thought it best to override the `announce` function from setuptools, to use the logging module solely in `commands.py`. Happy to change this.

Reply:
I see.  That's annoying.  I can't think of a cleaner solution at the moment.

Reply:
I think we have to verify that it's still possible to increase the output verbosity from the command-line with the setuptools verbosity flag